growl.lua

```diff
@@ -86,23 +86,15 @@ function main()
   SV:finish()
 end
 
-local function getProjectPathName()
-  local projectFileName = SV:getProject():getFileName()
-  if not projectFileName then return end
-
-  local projectName, projectDir
-  projectFileName = projectFileName:gsub("\\", "/")
-  projectDir, projectName = projectFileName:match("^(.*/)([^/]+)%.svp$")

```

Comment:
Does omitting this regexp fix the different charset issue? Or does this PR fix it?

Reply:
I am unable to reproduce the charset issue, but I suspect the regexp is not charset aware and this may help.
Instead of using the directory it uses the project name as given, so the settings are saved per project.